CHANGELOG.md

```diff
@@ -18,6 +19,41 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
+# r3.1
+
+## Release Notes
+
+This release contains the definition and documentation of
+* call-forwarding-signal v0.4.0-alpha.1
+
+The API definition(s) are based on
+* Commonalities v0.6.0
+* Identity and Consent Management v0.4.0

```

Comment:
I suppose you refer to the alpha versions here (r3.1 of each working group)?

Reply:
I'm sorry I'm not sure about the comment. The versions for Commonalities and ICM are different for each Release. Specifically,  6.0 and 4.0 are for r3.1. But this is clearly written in the document so I'm probably missing something :-)